src/main/java/com/google/sps/data/PlayerDatabase.java

```diff
@@ -63,18 +67,22 @@ public static Entity createEntityFromPlayer(Player player) {
     String email = player.getEmail();
     String imageID = player.getImageID();
     String currentPageID = player.getCurrentPageID();
+    int experiencePoints = player.getExperiencePoints();
+    int promotionThreshold = player.getPromotionThreshold();
```

Comment:
shouldn't the threshold be  gameStage  specific rather than a player attribute because every level on gameStage would have the same threshold depending on the level and that can be statically written. rather initializing player with a threshold that updates.